src/views/home-view.js

```diff
@@ -90,7 +90,7 @@ class HomeView extends ViewMixin(ReduxMixin(PolymerElement)) {
     }
 
     button:hover {
-        background-color: #007bbf;
+        background-color: #230871;
```

Comment:
<img width="1138" alt="image" src="https://github.com/FACT-Finder-Web-Components/documentation/assets/37622914/ebffb72a-f151-48f9-aae3-45e100cc9f5b">
this button previously had a hover effect. I temporarily set the hover color to the same as regular. Let me know if we can turn off the hover effect completely. Or maybe this button should keep its old colors?

Reply:
Good call on this one! I think it's only consistent to also apply the new colours to this button.
We can do without the hover effect.

Reply:
hover effect removed